ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -238,6 +230,14 @@ nnfw_open (const GstTensorFilterProperties * prop, void **private_data)
     goto session_exit;
   }
 
+  accelerator = nnfw_get_accelerator (pdata, prop->accl_str);
+  status = nnfw_set_available_backends (pdata->session, accelerator);
+  if (status != NNFW_STATUS_NO_ERROR) {
+    err = -EINVAL;
+    g_printerr ("Cannot set nnfw-runtime backend to %s\n", accelerator);
+    goto unalloc_exit;
```

Comment:
IMO, it should be `session_exit` instead of `unalloc_exit` since the session is already created by calling `nnfw_create_session()`.

Reply:
Yes. that has been a bug. It should've been session_exit even before this commit.

Reply:
updated